plugin/src/main/java/io/deephaven/plugin/type/ObjectTypeClassBase.java

```diff
@@ -25,32 +30,14 @@ public final String name() {
     }
 
     @Override
-    public final boolean isType(Object o) {
-        return clazz.isInstance(o);
+    public final boolean isType(Object object) {
+        return clazz.equals(object.getClass());
```

Comment:
Should we use reference equality? `.equals` can mask incompatibilities from different classloaders.

Reply:
Afaict, Class#equals is reference equality.

Reply:
I think you're right. Disregard, I guess.